lib/Controller/ApiController.php

```diff
@@ -206,6 +208,54 @@ public function export(array $migrators): DataResponse {
 			throw new OCSException('User migration operation already queued');
 		}
 
+		return $user;
+	}
+
+	/**
+	 * @NoAdminRequired
+	 * @NoSubAdminRequired
+	 *
+	 * @throws OCSException
+	 */
+	public function exportable(?array $migrators): DataResponse {
+		$user = $this->checkJobAndGetUser();
+
+		if (!is_null($migrators)) {
+			$this->checkMigrators($migrators);
+		}
+
+		try {
+			$size = $this->migrationService->estimateExportSize($user, $migrators);
+			// Convert to MiB and round to one significant digit after the decimal point
+			$roundedSize = round($size / 1024, 1);
+		} catch (UserMigrationException $e) {
+			throw new OCSException($e->getMessage());
+		}
+
+		try {
+			$this->migrationService->checkExportability($user, $migrators);
+		} catch (NotExportableException $e) {
+			$warning = $e->getMessage();
+		}
+
+		return new DataResponse([
+			'size' => $roundedSize,
```

Comment:
Maybe use a key like `estimated-size` to make it clear?
If I understood correctly this is returned by a get on /export, so it could be wrongly interpreted as information on the previous export rather than on the potential futur one.

Reply:
Updated. Since the `POST` on this endpoint deals with initiating a new export a `GET` would concern the same new export which makes sense to me